addons/pos_self_order/static/src/app/models/order.js

```diff
@@ -34,6 +36,8 @@ export class Order extends Reactive {
         this.lines = order.lines || [];
         this.tracking_number = order.tracking_number || null;
         this.take_away = typeof order.take_away === "boolean" ? order.take_away : null;
+        this.ticketCode = random5Chars(); // 5-digits alphanum code shown on the receipt
```

Comment:
Field that not in the backend should not be in the models, but if really needed put in under `//data` comment :)

Reply:
yes, i have to save it to the db